organizations.json

```diff
@@ -13,5 +13,19 @@
     "name": "Renzo",
     "identifier": "urn:oid:2.16.840.1.113883.2.4.6.1:13579246",
     "publicKey": "-----BEGIN PUBLIC KEY-----\nMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEApXmCBIrbPKY4sPin5m1jUFy9Ht9eil9kvxoUkMspQbKMTzAASgUbXQflDLzc1aJ5VpugaS3c0XVhkpxVtxH5Sd0Q1VFeIB28VmpCbcm8Gh7TwPwsZ0JBY4CL6P1dnJyO7ZRkOzppj03UMjGzqff6tiK6FU1Dcw5QK2XZiJJKmndOPvuwC0RoeaR1IHSU8Q4iOxiaR3OZBtcSw2oiLMxCwhgGhzfYu+geqZ0VmdCHG5w30f99S8i4qvJb16XRBy2g2p69NbWMbFOT4LxcRnxXAxH6VTiMRWra49W9anYLhT+2FWiJS6cu0V8A1Ro0MQV3nzUhfpZEnW+lKcO/Fp8RJQIDAQAB\n-----END PUBLIC KEY-----",
+  },
+  {
+    "name": "Huisarts Jan",
+    "identifier": "4111986",
+	"publicKey": "-----BEGIN PUBLIC KEY-----
```

Comment:
Can you put \n at each new line and make this one line?

Reply:
sorry about that, fixed it

Reply:
well, I actually mean the characters '\n' for each newline ;)

Reply:
i hipe its fine this time